services/app-web/src/pages/LinkYourRates/LinkYourRates.test.tsx

```diff
@@ -0,0 +1,115 @@
+            <Formik
+                initialValues={{ ratePreviouslySubmitted: '' }}
+                onSubmit={(values) => console.info('submitted', values)}
+            >
+                <form>
+                    <LinkYourRates />
+                </form>
+            </Formik>,
+            {
+                apolloProvider: {
+                    mocks: [
+                        fetchCurrentUserMock({
+                            statusCode: 200,
+                            user: mockValidStateUser(),
+                        }),
+                        indexRatesMockSuccess(),
+                    ],
+                },
+            }
+        )
+
+        await waitFor(() => {
+            expect(
+                screen.queryByText(
+                    'Was this rate certification included with another submission?'
+                )
+            ).toBeInTheDocument()
+        })
+    })
+
+    it('does not display dropdown menu if no is selected', async () => {
+        renderWithProviders(
+            <Formik
+                initialValues={{ ratePreviouslySubmitted: '' }}
+                onSubmit={(values) => console.info('submitted', values)}
+            >
+                <form>
+                    <LinkYourRates />
+                </form>
+            </Formik>,
+            {
+                apolloProvider: {
+                    mocks: [
+                        fetchCurrentUserMock({
+                            statusCode: 200,
+                            user: mockValidStateUser(),
+                        }),
+                        indexRatesMockSuccess(),
+                    ],
+                },
+            }
+        )
+
+        screen
+            .getByLabelText(
+                'No, this rate certification was not included with any other submissions'
+            )
+            .click()
+
+        await waitFor(() => {
+            expect(
+                screen.queryByText('Which rate certification was it?')
+            ).not.toBeInTheDocument()
+        })
+    })
+
+    it('displays dropdown menu if yes is selected', async () => {
+        renderWithProviders(
+            <Formik
+                initialValues={{ ratePreviouslySubmitted: '' }}
+                onSubmit={(values) => console.info('submitted', values)}
+            >
+                <form>
+                    <LinkYourRates />
+                </form>
+            </Formik>,
+            {
+                apolloProvider: {
+                    mocks: [
+                        fetchCurrentUserMock({
+                            statusCode: 200,
+                            user: mockValidStateUser(),
+                        }),
+                        indexRatesMockSuccess(),
+                    ],
+                },
+            }
+        )
+
+        screen
+            .getByLabelText(
+                'Yes, this rate certification is part of another submission'
+            )
+            .click()
```

Comment:
In general you should use the [userEvent](https://testing-library.com/docs/user-event/intro) library instead of "click". I think it handles actions better. 

For an example of this, see APIAccess.test.ts

you can pull the `user` out of the return from `renderWithProviders` then call `user.click(screen.getByLabelText(...))`

Calling click() directly on stuff is supposedly less user like and I suspect can lead to weird behavior in the end